### PR TITLE
Noa/copyright credit

### PR DESCRIPTION
**Pull request by noaresare**
This change makes sure we attribute the external contributions properly.

### CODE REVIEW

**Comment by noaresare** [comment]
ping @davidxia @rohansingh

File: NOTICE [hunk 1]
```diff
@@ -1,2 +1,14 @@
 Docker Client
-Copyright (c) 2014 Spotify AB
+This software was originally developed by Spotify. The portions originating
+from Spotify is copyright Spotify AB.
+
+Substantial additions to this project has been made by the following
```
Comment (davidxia) [inline NOTICE]: verb-noun agreement: "Substantial additions to this project has been" change "has" to "have"

File: NOTICE [hunk 2]
```diff
@@ -1,2 +1,14 @@
...
+contributions, who retain copyright in their contributions. For details
```
Comment (davidxia) [inline NOTICE]: "contributions, who retain copyright in their contributions": change the first "contributions" to "contributors"?

File: NOTICE [hunk 3]
```diff
@@ -1,2 +1,14 @@
...
+on the exent of each such contribution, see file headers and the git commit
```
Comment (davidxia) [inline NOTICE]: "exent" typo, should be "extent"

**Comment by davidxia** [comment]
@noaresare nice, thanks. see my comments above.

**Comment by noaresare** [comment]
Thank you. In other news, I should never write texts when in a hurry.

**Pull request merged**